src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -107,120 +154,326 @@ const CoverImageEditModal = ({
-                  ? dragProps.fileDropError
-                  : "Drag & drop image to upload"}
-              </p>
-              <p className="mt-4 text-gray-700 font-medium text-center">
-                No cover photo uploaded for this facility. <br />
-                Recommended aspect ratio for facility cover photo is 1:1.
-              </p>
-            </div>
-          )}
-
-          <div className="flex flex-col sm:flex-row p-4 gap-2">
-            <div>
-              <label className="rounded-lg bg-white py-2 px-4 text-primary-500 font-medium border border-primary-500 hover:text-primary-400 hover:border-primary-400 text-sm flex gap-1 items-center justify-center cursor-pointer transition-all">
-                <i className="fas fa-cloud-upload-alt mr-2"></i>Upload an image
-                <input
-                  title="changeFile"
-                  type="file"
-                  accept="image/*"
-                  className="hidden"
-                  onChange={onSelectFile}
-                />
-              </label>
-            </div>
-            <div className="sm:flex-1" />
-            <button
-              type="button"
-              className="rounded-lg bg-gray-100 hover:bg-gray-300 py-2 px-4 text-slate-600 hover:text-slate-800 font-medium text-sm flex gap-1 items-center justify-center  transition-all"
-              onClick={(e) => {
-                e.stopPropagation();
-                closeModal();
-                dragProps.setFileDropError("");
-              }}
-              disabled={isUploading}
-            >
-              Cancel
-            </button>
-            {facility.read_cover_image_url && (
+                {" "}
+                Open Camera
+              </button>
               <button
                 type="button"
-                className="rounded-lg bg-danger-500 py-2 px-4 text-white font-medium text-sm flex gap-1 items-center justify-center transition-all"
-                onClick={handleDelete}
+                className="rounded-lg bg-gray-100 hover:bg-gray-300 py-2 px-4 text-slate-600 hover:text-slate-800 font-medium text-sm flex gap-1 items-center justify-center  transition-all"
+                onClick={(e) => {
+                  e.stopPropagation();
+                  closeModal();
+                  dragProps.setFileDropError("");
+                }}
                 disabled={isUploading}
               >
-                Delete
+                Cancel
               </button>
-            )}
-            <button
-              type="button"
-              className="rounded-lg bg-primary-500 py-2 px-4 text-white font-medium hover:bg-primary-400 text-sm flex gap-3 items-center justify-center transition-all"
-              onClick={handleUpload}
-              disabled={isUploading}
-            >
-              {isUploading ? (
-                <i className="fa-solid fa-spinner animate-spin" />
+              {facility.read_cover_image_url && (
+                <button
```

Comment:
Replace with ButtonV2

Reply:
We can migrate the button in some other issue as it is not related to the feature @rithviknishad